group_vars/all.yaml

```
@@ -117,6 +117,10 @@ gxy_io_rewrites:
       - src: /GTN:ansible-galaxy
         dest: https://training.galaxyproject.org/training-material/short/ansible-galaxy
 
+  # TPV shared database
+  - src: /tpv/shared_db.yml
```

How about a simpler, shorter url? 

```suggestion
  - src: /tpv/db.yml
```

Even if not, I'd suggest doing `shared-db.yml` (hyphen vs underscore) as a preferred character for URLs.